profiles/calhoun/terraform/calhoun/vault.tf

```diff
@@ -0,0 +1,9 @@
+resource "vault_generic_secret" "endpoint" {
+  path = "${var.vault_path_prefix}/${var.service}/secrets/endpoint"
```

Comment:
I think I'd still prefer to call this "hostname" because it is the hostname part of the URL (it doesn't include a scheme or path)

Reply:
Mm good point